lib/runner.js

```diff
@@ -238,15 +238,6 @@ class Runner extends EventEmitter {
       }
     };
   }
-}
-
-/**
- * Wrapper for setImmediate, process.nextTick, or browser polyfill.
- *
- * @param {Function} fn
- * @private
- */
-Runner.immediately = global.setImmediate || process.nextTick;
```

Comment:
please move this back up to line ~250, a tiny part of me is worried these values might be changed somewhere in the next 1000 lines :D